src/pages/settings/following/index.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <ul class="author-list">
+    <template v-if="authorLikerIds.length">
+      <AuthorListItem
+        v-for="id in authorLikerIds"
+        :key="id"
+        :liker-id="id"
+      >
+        <template>
+          <button
+            class="btn btn--outlined btn--block"
+            @click="onClickActionButton(id)"
+          >{{ actionButtonText }}</button>
+        </template>
+      </AuthorListItem>
+    </template>
+    <div
+      v-else
+      class="text-gray-c text-center px-16 py-48"
+    >{{ emptyListText }}</div>
+  </ul>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+import AuthorListItem from '~/components/AuthorListItem';
+
+export default {
+  components: {
+    AuthorListItem,
+  },
+  computed: {
+    ...mapGetters(['getSubscribedAuthors', 'getUnsubscribedAuthors']),
+
+    authorLikerIds() {
+      if (this.$route.name === 'settings-following') {
```

Comment:
`this.$route.name === 'settings-following'`  should be abstracted to a computed property `is(Un)subscribeSetting`